### ISSUE TITLE
Magic bursting on react

### ISSUE THREAD

**Issue by Superdela** [SammehFFXI/FFXIAddons#22]
Cant figure out how to make it recognize skillchains, and well.. react.

**Comment by mac10688** [comment]
Yeah I've been trying to consider what it would take for this myself.

When a magic burst happens on a mob, how easy is it to get the id of the mob and react with a proper nuking spell?

I talked with the devs on discord and it seemed that it would be a bit difficult.
